setup.md

```diff
@@ -39,9 +39,19 @@ to download an installer package. Please note that
 JRE or JDK installed on your system, most distribution repositories will contain OpenJRE / OpenJDK packages. 
 Install the default version available from your distribution. For example, on Ubuntu/Debian: 
 `sudo apt install default-jre`.
-* OpenRefine does not support Internet Explorer. Please use [Firefox](https://www.mozilla.org/firefox/new/), 
+* OpenRefine does not support Internet Explorer. Please use [Firefox](https://www.mozilla.org/firefox/new/),
+[Microsoft Edge](https://www.microsoft.com/edge),
```

Comment:
This section could also be omitted, if https://docs.openrefine.org/manual/installing#compatible-browser had the links.

Reply:
That's been merged now